scripts/recipe_robot_lib/inspect.py

```diff
@@ -614,6 +614,29 @@ def inspect_disk_image(input_path, args, facts):
     else:
         facts["inspections"].append("disk_image")
 
+    # See if we can determine the download URL from the file metadata.
+    if "download_url" not in facts:
+        cmd = "/usr/bin/xattr -p com.apple.metadata:kMDItemWhereFroms \"%s\"" % input_path
```

Comment:
python has an matter module; no need to call out to xattr (which if you look, you will discover xattr _is_ a Python script!)